.github/workflows/copilot_deploy.yml

```diff
@@ -18,7 +18,7 @@ on:
         description: Run performance tests
       run_e2e_tests:
         required: false
-        default: true
+        default: false
```

Comment:
Think the default needs to be true, otherwise a trigger on push will never run the tests

Reply:
Amended